### ISSUE TITLE
./server test deletes database records

### ISSUE THREAD

**Issue by Manicben** [mozilla/ichnaea#506]
Using release 2.1.0.

After manually loading records into the location database (only into cell_gsm so far), then running ./server test, all records are removed. Had 7688936 records and after test script, 0 records.
Restarting containers does not cause this behaviour.

**Comment by hannosch** [comment]
Well, yes. The tests are integration tests running against the database they are connected to. They drop all tables, recreate the schema from scratch and test the database migration scripts. Same applies to the Redis instance, where the db contents are removed.

The `server` script is a script used only for developing the codebase itself. It's not meant to be used in a stage or production environment.

**Comment by Manicben** [comment]
I see, all makes sense.

On a side note, previous releases (pre 2.0) had a command for importing cell data (both cell and OCID), which was removed later. How does one now import, for example, the MLS full cell export csv?
I've tried importing via a pymysql script and abiding by the cell table structure, but geolocate requests still don't seem to work. With release 2.1.0 I get 500 Internal Server Error. So far I only import into cell_gsm, other tables are empty.

Any help would be appreciated, let me know if there's a better place to be asking these questions.
